scripts/bintray/deploy_deb.bash

```diff
@@ -1,44 +1,83 @@
 #!/bin/bash -x
 
 set -eEuo pipefail
-set -x
 
-# TODO:
-# This is a hacky, hacky script to keep going
-# 
+function show_help {
+    cat - <<HEREDOC
+Usage:
 
-function usage {
-    echo "Usage: ${0##*/} <USER> <PASSWORD> <VERSION> [COMPONENT]"
+    ${0##*/} -u USERNAME -p PASSWORD -v VERSION -r RPM_LIFECYCLE -c DEB_COMPONENT [-d]
+
+    where:
+        -d            - debug mode
+        DEB_COMPONENT - component within the Debian repository;
```

Comment:
The documentation for RPM_LIFECYCLE should be here too

Reply:
Not really. Deploy_deb and deploy_rpm share all other parameters **but** the lifecycle / component.

I thought to name it the same (lifecycle) but then, their values may be different, semantics are a bit different as well.